etcd/etcd-live-http-to-https-migration.md

```diff
@@ -142,24 +132,41 @@ etcdctl member update 50da8780fd6c8919 https://172.16.0.103:2380
 etcdctl member update 81901418ed658b78 https://172.16.0.101:2380
 ```
 
-You have to run generated commands **except last one** on one of the member nodes and check etcd cluster health:
+On any one etcd cluster member, run each of the printed commands -- *except the last one*. We will change the peer URL for the last etcd cluster member only after completing any proxy configuration below.
```

Comment:
If we switch URLs of all etcd members, and don't reconfigure them in `proxy-refresh-interval` (30 sec) time period, etcd proxy will update these urls and will try to connect to new URLs which are not available yet. That will break cluster. If we keep one etcd member to listen insecure port - etcd proxy will still work.

We cannot accomplish this procedure in 30 secs. Or even if we increase this interval - something could get wrong and we'll have troubles.